### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by translatewiki**
Translation updates

### CODE REVIEW

File: core/src/main/res/values-ha/strings.xml [hunk 1]
```diff
@@ -238,6 +239,7 @@
   <string name="severe_loss_error">Kuskure mai tsanani! Gwada Kashe/Sake kunna WiFi P2P</string>
   <string name="connection_failed">Haɗin ya gaza</string>
   <string name="permission_rationale_location">Android na buƙatar izinin wuri don ba da damar ƙa’idar ta gano na’urorin tsara</string>
+  <string name="permission_rationale_nearby">Android na buƙatar izinin na\'urorin kusa don ba da damar ƙa\'idar ta gano na\'urorin tsara</string>
```
Comment (MohitMaliFtechiz) [inline core/src/main/res/values-ha/strings.xml:242]: Like here.

File: core/src/main/res/values-dag/strings.xml [hunk 2]
```diff
@@ -247,6 +260,7 @@
   <string name="no_devices_found">Bimbuɣisirigu bi nya. Dihimi vihima ka di labi niŋ yaha.</string>
   <string name="files_for_transfer">FASARA NIMA DIN YƐN ZAŊ KPƐHI</string>
   <string name="preparing_files">Maani fasara nima niŋdi ni di zaŋ kpɛhi</string>
+  <string name="performing_handshake">Nuu\'tibu niŋbu</string>
```
Comment (MohitMaliFtechiz) [inline core/src/main/res/values-dag/strings.xml:263]: Here

File: core/src/main/res/values-ig/strings.xml [hunk 3]
```diff
@@ -335,4 +335,8 @@
   <string name="your_device_name_message">Aha ngwaọrụ gị ga-apụta ebe a.</string>
   <string name="nearby_devices_list_message">Ebe a, ị ga-ahụ ndepụta nke ngwaọrụ dị nso. Kpatụ aha ngwaọrụ ka ịmalite nyefe faịlụ.</string>
   <string name="transfer_zim_files_list_message">Ebe a, ị ga-ahụ ndepụta nke faịlụ ZIM dị maka mbufe.</string>
+  <string name="storage_selection_dialog_accessibility_description">Họrọ nchekwa maka nbudata faịlụ ZIM.</string>
+  <string name="zim_file_content_description">faịlụ ZIM nke nwere ọdịnaya ọgụgụ</string>
+  <string name="select_language_content_description">Ịhọrọ asụsụ a ga-ebute ụzọ igosipụta akwụkwọ enwere ike ibudata n\'asụsụ ahụ dị n\'elu.</string>
```
Comment (MohitMaliFtechiz) [inline core/src/main/res/values-ig/strings.xml:340]: here.

**Review by MohitMaliFtechiz — CHANGES_REQUESTED** [review]
@kelson42 Some new changes contain the `TypographyQuotes` errors so I have placed a fix for those on the upstream. They will come in the upcoming update of TW, we should wait for those changes. Apart from this, other changes are LGTM.

File: core/src/main/res/values-dag/strings.xml (hunk 2 above)
Comment (kelson42) [reply core/src/main/res/values-dag/strings.xml:263]: Please do the change upstream

File: core/src/main/res/values-ig/strings.xml (hunk 3 above)
Comment (kelson42) [reply core/src/main/res/values-ig/strings.xml:340]: Please do the change upstream if they are technical

Comment (MohitMaliFtechiz) [reply core/src/main/res/values-ig/strings.xml:340]: @kelson42 I have made the changes on upstream.

**Review by MohitMaliFtechiz — APPROVED** [review]
@kelson42 All the changes have come that fix the TypographyQuotes error. I have reviewed the PR and now it's LGTM.

**Pull request merged**